packages/editor-ui/src/components/canvas/Canvas.vue

```diff
@@ -122,7 +122,7 @@ const classes = computed(() => ({
 
 const disableKeyBindings = computed(() => !props.keyBindings);
 
-const panningKeyCode = 'Shift';
+const panningKeyCode = ' ';
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values#whitespace_keys

Reply:
👍 Could even add it as a comment to the code

Reply:
done